### ISSUE TITLE
How to lazy load background images?

### ISSUE THREAD

**Issue by Tjitse-E** [magefan/module-lazyload#10]
Hi,

Is it possible to lazy load a CSS background image? Like this for example: https://github.com/verlok/lazyload#lazy-background-image

**Comment by magefan** [comment]
Please use this fix https://github.com/magefan/module-lazyload/commit/c4f379184669956b6b50c1df0515f8644afbfe22 for [Magento 2 Image Lazy Load](https://magefan.com/magento-2-image-lazy-load-extension)
Then you will be able to lazy load div background, example:
```
<div data-original="https://opt.demo.magefan.top/media/catalog/product/cache/f073062f50e48eb0f0998593e568d857/w/j/wj11-blue_main.jpg" 
style="width:240px; height:300px;"></div>
```

**Comment by Tjitse-E** [comment]
This seems to work, thanks! Could you create a new release @magefan?

**Comment by magefan** [comment]
@Tjitse-E , we have released v2.0.7
